assets/css/global.css

```diff
@@ -48,7 +48,7 @@
     */
     --text: var(--polar-night-dark);
     --text-muted: var(--polar-night-very-light);
-    --background: url('/bg/light.svg');
+    --background: url('css/bg/light.svg');
```

Comment:
Sorry, that still won't work. Here's how it's done:
![image](https://user-images.githubusercontent.com/60521810/155311312-6542bd72-fb9a-4fff-8c5f-fa346a9d224d.png)